Planner/app/src/main/java/rdatu/android/cyscorpions/com/projectplanner/view/ListPlannerFragment.java

```diff
@@ -0,0 +1,324 @@
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View v = super.onCreateView(inflater, container, savedInstanceState);
+        return v;
+    }
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        ListView lv = getListView();
+        registerForContextMenu(lv);
+
+        super.onActivityCreated(savedInstanceState);
+    }
+
+    protected final void onNextDay() {
+
+        if (mCalendar.get(Calendar.DAY_OF_MONTH) == mCalendar.getActualMaximum(Calendar.DAY_OF_MONTH)) {
+            mCalendar.add(Calendar.MONTH, 1);
+            mCalendar.set(Calendar.DAY_OF_MONTH, 1);
+        } else {
+            mCalendar.add(Calendar.DAY_OF_MONTH, 1);
+        }
+        try {
+            mCallbacks.onListUpdate(getStringDate());
+        } catch (Exception e) {
+            Log.e("Planner", "Error", e);
+        }
+
+
+        try {
+            getListView().invalidateViews();
+        } catch (Exception e) {
+
+        }
+    }
+
+
+    protected final void onPreviousDay() {
+
+        if (mCalendar.get(Calendar.DAY_OF_MONTH) == mCalendar.getActualMinimum(Calendar.DAY_OF_MONTH)) {
+            mCalendar.add(Calendar.MONTH, -1);
+            mCalendar.set(Calendar.DAY_OF_MONTH, mCalendar.getActualMaximum(Calendar.DAY_OF_MONTH));
+        } else {
+            mCalendar.add(Calendar.DAY_OF_MONTH, -1);
+        }
+
+        try {
+            mCallbacks.onListUpdate(getStringDate());
+        } catch (Exception e) {
+            Log.e("Planner", "Error", e);
+        }
+        try {
+            getListView().invalidateViews();
+        } catch (Exception e) {
+        }
+    }
+
+    private String getStringDate() {
+        SimpleDateFormat df = new SimpleDateFormat("MMM-dd-yyyy");
+        return df.format(mCalendar.getTime());
+    }
+
+    @Override
+    public void onListItemClick(ListView l, View v, int position, long id) {
+        String name, place, descr, priority, task;
+        Tasks tasks;
+        String time = ((TextView) v.findViewById(R.id.time_slot)).getText().toString();
+        mTaskManager.getSpecificTask(getActivity().getTitle().toString(), time);
+        tasks = mTaskManager.getLoadedTask();
+        if (tasks != null) {
+            name = tasks.getTaskName();
+            place = tasks.getPlace();
+            descr = tasks.getDescription();
+            priority = tasks.getPriority();
+            task = "edit";
+        } else {
+            name = "";
+            place = "";
+            descr = "";
+            priority = "LOW";
+            task = "FUNCTION";
+        }
+        mCallbacks.onTimeSlotSelected(time, getActivity().getTitle().toString(), name, place, descr, priority, task);
+    }
+
+    public void onResume() {
+        super.onResume();
+        getListView().invalidateViews();
+    }
+
+    @Override
+    public void onDateChanged(String date) {
+        //Not Used
+    }
```

Comment:
Ma'am I can't remove that po since it will result to an error I need to implement this po even if it is blank. since I implemented it from an interface po.